index.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
     type: 'apply-for-compensation',
-    version: '1.5.0',
+    version: '1.6.1',
```

Comment:
what info am I missing here? why is there 2 bumps?

Reply:
As confirmed in the chat, this is the correct version!